aws/resource_aws_redshift_cluster_test.go

```diff
@@ -320,6 +320,37 @@ func TestAccAWSRedshiftCluster_tags(t *testing.T) {
 	})
 }
 
+func TestAccAWSRedshiftCluster_forceNewUsername(t *testing.T) {
+	var v redshift.Cluster
```

Comment:
_minor_ as you're storing this you could compare the old and new clusters to ensure it's been recreated?